src/TriggerBinding/SqlTriggerBindingProvider.cs

```diff
@@ -22,18 +22,20 @@ internal sealed class SqlTriggerBindingProvider : ITriggerBindingProvider
         private readonly IConfiguration _configuration;
         private readonly IHostIdProvider _hostIdProvider;
         private readonly ILogger _logger;
+        private readonly SqlOptions _sqlOptions;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SqlTriggerBindingProvider"/> class.
         /// </summary>
         /// <param name="configuration">Configuration to retrieve settings from</param>
         /// <param name="hostIdProvider">Provider of unique host identifier</param>
         /// <param name="loggerFactory">Used to create logger instance</param>
-        public SqlTriggerBindingProvider(IConfiguration configuration, IHostIdProvider hostIdProvider, ILoggerFactory loggerFactory)
+        /// <param name="sqlOptions"></param>
+        public SqlTriggerBindingProvider(IConfiguration configuration, IHostIdProvider hostIdProvider, ILoggerFactory loggerFactory, Microsoft.Extensions.Options.IOptions<SqlOptions> sqlOptions)
         {
             this._configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
             this._hostIdProvider = hostIdProvider ?? throw new ArgumentNullException(nameof(hostIdProvider));
-
+            this._sqlOptions = sqlOptions?.Value ?? new SqlOptions();
```

Comment:
I don't think you should ever be newing up the options yourself like this - you always want options to be hydrated by the configuration system